frontend/viewer/src/project/ProjectDropdown.svelte

```diff
@@ -107,7 +107,7 @@
           </Command.Loading>
         {:else}
           <Command.Empty>{$t`No Dictionaries found`}</Command.Empty>
-          {#each projectsResource.current ?? [] as project (project.id)}
+          {#each projectsResource.current ?? [] as project (`${project.code}-${project.crdt}`)}
```

Comment:
There are also three instances of `{#each projects as project (project.id)}`, one of `{#each projects as proj (proj.id)}`, and one of `{#each relatedProjects.current as proj (proj.id)}` in `frontend/src/`. Might any of those need the same treatment?